src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js

```diff
@@ -146,6 +147,7 @@ class OrderTemplatesEditor extends Component {
     } = this.props;
     const { sections } = this.state;
     const orderFormat = formValues.orderFormat;
+    const estimatedPrice = calculateEstimatedPrice(formValues);
```

Comment:
I'm not sure that `calculateEstimatedPrice(formValues)` is a generic fn. `formValues` will be different in ui-invoice from ui-orders, no?

Reply:
yes. but I don't use this function on invoice and don't moved this function to stripes-acq-components